unittests/test_buildsystems.py

```diff
@@ -235,3 +235,20 @@ def test_compiler_pick(lang):
     }
     assert ([f'{compilers[lang]} {build_system.srcfile} -o foo.x'] ==
             build_system.emit_build_commands(ProgEnvironment('testenv')))
+
+
+def test_singlesource_unknown_language():
+    build_system = bs.SingleSource()
+    build_system.cc = 'cc'
+    build_system.cxx = 'CC'
+    build_system.ftn = 'ftn'
+    build_system.nvcc = 'nvcc'
```

Comment:
Why do you need to set all these?

Reply:
I will remove. I put there to have the compilers in place.